Time off tracker WEB/Startup.cs

```diff
@@ -0,0 +1,44 @@
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+
+namespace Time_off_tracker_WEB
+{
+    public class Startup
+    {
+
+        public Startup(IConfiguration configuration) 
+        {
+            Configuration = configuration;
+        }
+
+        public IConfiguration Configuration { get; }
+
+        public void ConfigureServices(IServiceCollection services)
+        {
+            services.AddControllers(mvcOtions =>
+            {
+                mvcOtions.EnableEndpointRouting = false;
+            });
+        }
+
+        public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
+        {
+            if (env.IsDevelopment())
+            {
+                app.UseDeveloperExceptionPage();
+            }
+
+            app.UseHttpsRedirection();
+
+            app.UseRouting();
+
+            //CORS setting
+            app.UseCors();
```

Comment:
did u test this? Are you able to  do cross-domain requests?

Reply:
I thought that we will set up a CORS options later.
Add builder options. It is working now. More detailed options will be installed as needed.
![image](https://user-images.githubusercontent.com/70018123/95116409-a7906100-074f-11eb-9371-d104149001da.png)